tokens/blockmirmain/MIR.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "MIR",
+  "symbol": "MIR",
+  "account": "blockmirmain",
+  "precision": 4,
+  "issuer": "blockmirmain",
+  "logo": "https://raw.githubusercontent.com/BlockABC/eos-tokens/master/tokens/eosblackteam/MIR.png",

```

Comment:
This is not right.

Reply:
The mistake was corrected.
Please register.